Microsoft.Dynamics365.UIAutomation.Api.UCI/WebClient.cs

```diff
@@ -3344,6 +3345,54 @@ internal BrowserCommandResult<bool> SelectLookupNewButton()
             });
         }
 
+        internal BrowserCommandResult<IReadOnlyCollection<FormNotification>> GetFormNotifications()
+        {
+            return Execute(GetOptions($"Get all form notifications"), driver =>
+            {
+                List<FormNotification> notifications = new List<FormNotification>();
+
+                if (!driver.TryFindElement(By.XPath(AppElements.Xpath[AppReference.Entity.FormNotifcationBar]),
+                    out var notificationBar))
+                {
+                    return notifications;
+                }
+
+                if (notificationBar.TryFindElement(By.XPath(AppElements.Xpath[AppReference.Entity.FormNotifcationExpandButton]), out var expandButton))
+                {

```

Comment:
Add a boolean variable, it is more readeable

Reply:
Also added some additional comments to make it more clear.